go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/openshift/api v0.0.0-20231129134630-a782d1c1541c
 	github.com/openshift/build-machinery-go v0.0.0-20230228230858-4cd708338479
 	github.com/openshift/client-go v0.0.0-20230926161409-848405da69e1
-	github.com/openshift/library-go v0.0.0-20240306175506-206f66c55642
+	github.com/openshift/library-go v0.0.0-20240403193518-3e3d5f9cf7a4
```

Comment:
ok, pinned to https://github.com/openshift/library-go/commit/3e3d5f9cf7a401348d840e63276766bc2c440dd7 which belongs to 4.15 branch.